pkg/restore/restore.go

```diff
@@ -454,17 +456,27 @@ func (ctx *context) execute() (Result, Result) {
 		}
 	}
 
-	// TODO timeout?
-	ctx.log.Debug("Waiting on global wait group")
-	waitErrs := ctx.globalWaitGroup.Wait()
-	ctx.log.Debug("Done waiting on global wait group")
+	ctx.log.Info("Waiting for all restic restores to complete")
```

Comment:
suggest moving this log message into the `go func` that is doing the waiting.

Reply:
done